src/services/textChanges.ts

```diff
@@ -337,8 +337,30 @@ namespace ts.textChanges {
             return this.replaceWithSingle(sourceFile, startPosition, startPosition, newNode, options);
         }
 
-        public insertNodeAfter(sourceFile: SourceFile, after: Node, newNode: Node, options: InsertNodeOptions & ConfigurableEnd = {}) {
-            if ((isStatementButNotDeclaration(after)) ||
+        public insertNodeAtConstructorStart(sourceFile: SourceFile, ctr: ConstructorDeclaration, newStatement: Statement, newLineCharacter: string): void {
+            if (ctr.body.statements.length === 0) {
+                this.replaceNode(sourceFile, ctr.body, createBlock([newStatement], /*multiLine*/ true), { useNonAdjustedEndPosition: true });
+            }
+            else {
+                this.insertNodeAfter(sourceFile, getOpenBrace(ctr, sourceFile), newStatement, { suffix: newLineCharacter });

```

Comment:
Is this different from inserting it before the first statement?

Reply:
If there's no linebreak after the brace, the new statement will appear on the same line?

Reply:
Yeah, failed to account for single-line constructors with statements in them. We should probably just make it multiline if it isn't already.